acceptance-tests/run-tests

```diff
@@ -66,24 +65,23 @@ setArgsOverrides(
 (async () => {
   try {
     const testRunner = new Jasmine();
+    testRunner.exitOnCompletion = false;
     testRunner.loadConfigFile(path.join(__dirname, 'jasmine.json'));
     testRunner.addReporter(reporter);
-    testRunner.onComplete(function(passed) {
-      rimraf.sync(CONFIG_FILE_PATH);
-      process.exit(passed ? 0 : 1);
-    });
-    const cliPath = path.join(process.cwd(), '../packages/cli/bin/hs');
-    const cli = cmd.create(cliPath, '.');
-    global.cli = cli;
+
     global.config = getTestConfig();
+    global.cli = cmd.createCli(global.config.cliPath);
 
-    await cli.execute(
-      ['init', `--c="${CONFIG_FILE_PATH}"`],
-      [cmd.ENTER, config.personalAccessKey, cmd.ENTER]
+    await global.cli.execute(
+      ['init', `--c="${CONFIG_FILE_NAME}"`],
+      [cmd.ENTER, global.config.personalAccessKey, cmd.ENTER, cmd.ENTER]
```

Comment:
This was the main issue. It admittedly took me pretty long to figure this out since the rest of the run-tests logic is annoying to debug. We needed to hit "enter" one more time to input the name for the account.